cmd/root.go

```diff
@@ -126,6 +127,7 @@ func init() {
 	rootCmd.PersistentFlags().IntVarP(&optMax, "max", "m", optMax, "Maximum length of string, keep on abbreviating while the string is longer than this limit")
 	rootCmd.PersistentFlags().BoolVarP(&optFrmFront, "from-front", "", optFrmFront, "Shorten from the front")
 	rootCmd.PersistentFlags().StringVarP(&optStrategy, "strategy", "", optStrategy, "Abbreviation strategy")
+	rootCmd.PersistentFlags().BoolVarP(&optRemoveStopwords, "stopword", "", optRemoveStopwords, "Remove stopwords from abbreviation")
```

Comment:
Could this option be called `no-stopwords`? I've tried to keep with GNU style options in this codebase.

Reply:
Okay! I've renamed it to `no-stopwords`